lib/spack/spack/ci/common.py

```diff
@@ -351,7 +351,7 @@ def report_skipped(self, spec: spack.spec.Spec, report_dir: str, reason: Optiona
         configuration = CDashConfiguration(
             upload_url=self.upload_url,
             packages=[spec.name],
-            build=self.build_name,
+            build=self.build_name(),
```

Comment:
Line 220's use of the method also needs `()`.

Reply:
should be taken care of now, thanks for pointing this out @tldahlgren